tests/AdminPanel/Symfony/AdminBundle/Tests/Functional/ListPageTest.php

```diff
@@ -244,4 +244,18 @@ public function test_change_list_elements_number_for_dbal_driver()
             ->shouldHaveElementsOnTheList(1)
         ;
     }
+
+    public function test_delete_batch_action()
+    {
+        $this->dbContext->createUser('l3l0');
+        $this->dbContext->createUser('otherUser');
+        $this->dbContext->createUser('niflheim');
+
+        (new ListPage($this->client))
+            ->open()
+            ->shouldHaveElementsOnTheList(3)
+            ->batchDeleteElements([1, 3])
+            ->shouldHaveElementsOnTheList(1)
```

Comment:
could you also check here if user that left is `otherUser`? (if that's what this test is testing :P)